src/components/Admin/Admin.js

```diff
@@ -93,12 +116,19 @@ class Admin extends Component {
   };
 
   render() {
-    const tabStyle = {
-      width: '100%',
-      animated: false,
-      textAlign: 'left',
-      display: 'inline-block',
-    };
+    let MobileView = isMobileView();
+
+    const { skillStats } = this.state;
+    const {
+      totalSkills = 0,
+      reviewedSkills = 0,
+      nonReviewedSkills = 0,
+    } = skillStats;
+    const totalSkillsCount = skillStats && totalSkills ? totalSkills : 0;
```

Comment:
Why are these 3 declarations required now. The statement I suggested initialises it to 0 if not present.